README.md

```diff
@@ -1,5 +1,60 @@
-# FCC Test Suite Template
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
```

Comment:
You remove this message, it is meaningless by now

Reply:
- [ ] Kindly make sure to deliver politely, and well-explained review comments so that students understand the point 🙏 😄 
Ex:
Instead of your comment, it'd be something like: "I don't see any use of this line, don't you think it's better if you remove it? 😄 This would be better for the professionalism of the repo and the way it looks 🙏"

Reply:
thanks,

Reply:
- [ ] Dear @Mucyosoda, I think you should make the changes requested by the reviewer pointed out in the above comment. Maybe, thanks, is not the right place here.